src/plugins/Analysis/ReactionFilter/DReaction_factory_ReactionFilter.h

```diff
@@ -32,6 +32,10 @@ class DReaction_factory_ReactionFilter : public jana::JFactory<DReaction>
 		{
 			// This is so that the created DReaction objects persist throughout the life of the program instead of being cleared each event. 
 			SetFactoryFlag(PERSISTANT);
+			
```

Comment:
These options would apply to all reactions in an analysis launch, right?  Would it be useful/practical to set these as command line flags that could be specified for individual reactions to reduce the file size?

Reply:
I was thinking of implementing those options in the future.   The general philosophy has been to apply common loose selections on an analysis launch which don't appreciably reject backgrounds, so I took that as the starting point.